src/pages/Events/Events.tsx

```diff
@@ -67,9 +74,19 @@ const Events = () => {
     setDbData(data as Event[]);
   };
 
+  useEffect(() => {
```

Comment:
This can be simplified by removing the useEffect, and state. 

These can just be variables that get assigned when the component gets updated. At the moment by having it in state you are adding additional re-renders that I do not think are necessary. 

```tsx

// ASSIGN VARIABLES NO STATE OR USE EFFECT
  const today = new Date();
  const date = format(today, "EEEE, do MMMM yyyy");
  
  // GIVE VARIABLES TO COMPONENTS RATHER THEN STATE
  
        <Header
        title="Events"
        subTitle={`MADE MY MAKERS STUDIO TOUR`}
        date={date}
      />
```